pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -369,13 +376,16 @@ def training_step(self, *args, **kwargs):
         return self.model(*args, **kwargs)
 
     def validation_step(self, *args, **kwargs):
-        return self.model(*args, **kwargs)
+        if isinstance(self.model, DistributedDataParallel):
+            return self.model(*args, **kwargs)
+        else:
+            return self.lightning_module.validation_step(*args, **kwargs)
 
     def test_step(self, *args, **kwargs):
-        return self.model(*args, **kwargs)
+        return self.lightning_module.test_step(*args, **kwargs)
 
     def predict_step(self, *args, **kwargs):
-        return self.model(*args, **kwargs)
+        return self.lightning_module.predict_step(*args, **kwargs)
```

Comment:
Are those changes required as we should  have `self.model == self.lightning_module` there, no ? 
Or is it for improving readability ?